std/socket.d

```diff
@@ -3522,8 +3522,8 @@ class UdpSocket: Socket
             @property @trusted Address localAddress() { assert(0); }
             @trusted ptrdiff_t send(const(void)[] buf, SocketFlags flags) { assert(0); }
             @safe ptrdiff_t send(const(void)[] buf) { assert(0); }
-            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, Address to) { assert(0); }
-            @safe ptrdiff_t sendTo(const(void)[] buf, Address to) { assert(0); }
+            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, in Address to) { assert(0); }
+            @safe ptrdiff_t sendTo(const(void)[] buf, in Address to) { assert(0); }
```

Comment:
> Its pretty prolific having to remove const all the way through my stack because one method was missing it :/ - by mistake?

std.socket is not const-aware because it predates `const`, and nobody updated it to support `const` until it was too late to change anything.

My suggestion is to accept for the moment that std.socket is an old module due for replacement, and to add a cast in your network code.